providers/tentacle.rb

```diff
@@ -281,3 +280,17 @@ def verify_roles(roles)
 def verify_environment(environment)
   raise 'Environment is required in order to register a Tentacle with Octopus Deploy Server' unless environment
 end
+
+def verify_port(port,polling)
```

Comment:
I would call this resolve_port and put it in the tentacle library file.  Also I would pass in actual true or false value and just use the fancy_boolean for powershell formatting only.

```
def resolve_port(polling, port)
  port || default_port(polling)
end

def default_port(polling) 
  if polling
    10_943
  else
    10_933
  end
end 
```